lib/app/person.ex

```diff
@@ -0,0 +1,39 @@
+defmodule App.Person do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias App.{Item, Repo, Tag}
+  alias __MODULE__
+  # https://hexdocs.pm/phoenix/Phoenix.Param.html
+  @derive {Phoenix.Param, key: :person_id}
+
+  @primary_key {:person_id, :id, autogenerate: false}
+  schema "people" do
+    field :name, :string
+
+    has_many :items, Item, foreign_key: :person_id
+    has_many :tags, Tag, foreign_key: :person_id
+    timestamps()
+  end
+
+  @doc false
+  def changeset(person, attrs \\ %{}) do
+    person
+    |> cast(attrs, [:person_id, :name])
+    |> validate_required([:person_id, :name])
+    |> unique_constraint(:name, name: :people_name_index)
```

Comment:
Can't say I understand why the `:name` should be unique ... the `:person_id`, sure, but name ...? 🤷‍♂️

Reply:
I'm thinking of using the name later on in the UI to select people to add to groups. If the name is not unique we won't be able to distinguish between users. Maybe it should be called `username`.